web_responsive_pwa/controllers/main.py

```diff
@@ -0,0 +1,25 @@
+from odoo.http import request, Controller, route
+
+
+class PWA(Controller):
+
+    @route('/service-worker.js', type='http', auth="public")
+    def service_worker(self):
+        qweb = request.env['ir.qweb'].sudo()
+        assets = qweb._get_asset_nodes("web.assets_common", {}, True, True)
```

Comment:
BTW why are we just caching these and not `web.assets_backend` for instance?

Reply:
Right, added